lib/cocoapods/target/build_settings.rb

```diff
@@ -1246,6 +1292,17 @@ def user_target_xcconfig_values_by_consumer_by_key
           merged_xcconfigs(user_target_xcconfig_values_by_consumer_by_key, :user_target_xcconfig)
         end
 
+        # @see BuildSettings#should_apply_xctunwrap_fix?
+        def should_apply_xctunwrap_fix?
```

Comment:
make this a `define_build_settings_method, :memoized => true` so it doesn't need to be re-computed every time it gets called